src/components/Navibar.js

```diff
@@ -26,7 +27,7 @@ export default class Navibar extends Component {
         return (
             <div>
                 <Navbar color="inverse" light expand="md" className = "nav-bar" >
-                    <NavbarBrand href={Home}>Our simple <br />logo here</NavbarBrand>
+                <NavbarBrand href={Home}><img src={logo} alt="logo"  style={{width: '200px', height: '100px'}}/></NavbarBrand>
```

Comment:
example  = `<img src='./images/AgroeasyLogo.png' ... />`

Reply:
No inline-styles. Css should be done in home.css

Reply:
It will be a more good modularity practice for each component to have its own css file

Reply:
@EngrEric yes each component should basically have its own css file, but @Simpcyclassy do not worry about that for now. i will make those changes

Reply:
OK @Way2nnadi